types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -33,4 +34,5 @@ export interface ChatItemTargetText extends BaseChatItemProps {
 export interface BaseChatItemComponentProps {
   item: ChatSchema;
   index: number;
+  type?: 'ANONYMOUS' | 'SIGNED';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `type` property has been added to the `BaseChatItemComponentProps` interface. Make sure that all components using this interface are updated to include this new property where needed.
```diff
+  type?: 'ANONYMOUS' | 'SIGNED';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->